pkgs/applications/version-management/datalad/default.nix

```diff
@@ -2,13 +2,13 @@
 
 python3.pkgs.buildPythonApplication rec {
   pname = "datalad";
-  version = "1.0.2";
+  version = "1.1.1";
```

Comment:
It's necessary? I think maybe wait this PR https://github.com/NixOS/nixpkgs/pull/329398 ?

Reply:
yes. Without this update (as stated in the description), it fails to build (due to boto vs boto3 and python3.12)